meta-leda-components/recipes-sdv/eclipse-leda/classes/kanto-auto-deployer.bbclass

```diff
@@ -11,4 +11,8 @@
 # * SPDX-License-Identifier: Apache-2.0
 # ********************************************************************************/
 #
+
 KANTO_MANIFESTS_LOCAL_DIR ??= "/var/containers/manifests"
+
+KANTO_MANIFESTS_DIR ??= "${KANTO_MANIFESTS_LOCAL_DIR}"
+KANTO_MANIFESTS_DIR[doc] = "The location path of Kanto Container Management deployment descriptors to autodeploy"
```

Comment:
KANTO_MANIFESTS_DIR ??= "/data/${KANTO_MANIFESTS_LOCAL_DIR}"

Reply:
No, the "/data" prefix is specific to the Leda Distribution and configured in the local.conf (or respective in one of the kas configurations).

This is just the default definition, so that by default it will be in "/var/.,.."
KANTO_MANIFESTS_DIR ??= "${KANTO_MANIFESTS_LOCAL_DIR}"

By overriding KANTO_MANIFESTS_DIR in the Leda Distro, it will be set to "/data/..."